src/SignalR/clients/java/signalr/core/src/main/java/com/microsoft/signalr/HttpHubConnectionBuilder.java

```diff
@@ -23,6 +23,8 @@ public class HttpHubConnectionBuilder {
     private Map<String, String> headers;
     private TransportEnum transportEnum;
     private Action1<OkHttpClient.Builder> configureBuilder;
+    private Long serverTimeout;
```

Comment:
Why use the object `Long` instead of the primitive `long`? It also adds unnecessary `Object.nonNull` code to `HubConnection` and we're storing it as `long` anyways.

Reply:
The reason for that is the following. 
If I used primitive `long` and `withServerTimeout` wasn't called in `HubConnectionBuilder` then default value 0 would be passed to `HubConnection` constructor and there would be a need to check that `serverTimeout` is not 0. But what if there would be a case when you want to set `serverTimeout` to 0. That is why I decided to use `Long` so that default value would be `null` and I would check for `null` in the `HubConnection` constructor.

Reply:
Give the values a default so they won't be 0 if not set.

Reply:
Done